README.md

```diff
@@ -0,0 +1,71 @@
+## salt-rekey
+
+This is a script designed to quickly re-key Salt minions. It was written
+originally as a part of the mitigation efforts for CVE-2020-11651 and
+CVE-2020-11652 but it can be used in any scenario in which all minions
+connected to a salt-master should be forced to re-generate their keys
+and re-connect.
+
+For background information on Salt's security model and the role of
+keys, please see this document:
+
+https://get.saltstack.com/rs/304-PHQ-615/images/SaltStack-Trust_Overview-of-SaltStack-security-model-white-paper.pdf
+
+## Why
+
+One may wish to rekey minions in any event where the key(s) of the minion
+or of the master can no longer be trusted, as in the case of a security
+breach.
+
+## Disclaimer
+
+This script is provided without any fitness or warranty. It may cause
+minions to be disconnected from the Salt master and fail to reconnect,
+forcing manuel intervention.
+
+## Installation
+
+To pull this project down for use on a Salt Master, you need `git`.
+Use the following command to download the script for use:
+
+`git clone git@github.com:cachedout/salt-rekey.git`
```

Comment:
````suggestion
```
git clone git@github.com:dwoz/salt-rekey.git
```
````

* Main thing above is `dwoz`, the formatting is an additional suggestion.

Might need to offer the alternative as well:

```
git clone https://github.com/dwoz/salt-rekey.git
```